server/src/models/product/product.controller.ts

```diff
@@ -140,7 +140,7 @@ export class ProductController {
     return this.boardService.deleteBoardContent(questionId, req.user!.id);
   }
 
-  @Get('/keywords')
+  @Get('/keywords/:query')
   getKeywords(@Param('query') query: string): Promise<ProductIdAndTitle[]> {
```

Comment:
```suggestion
  @Get('/keywords')
  getKeywords(@Query('query') query: string): Promise<ProductIdAndTitle[]> {
```

resource가 아니고 resource의 일부분 같은 느낌이라서 query로 바꿔주세요~